pmpro-cancel-on-next-payment-date.php

```diff
@@ -141,7 +150,7 @@ function pmproconpd_gettext_cancel_text($translated_text, $text, $domain) {
 
   if(($domain == "pmpro" || $domain == "paid-memberships-pro") && $text == "Your membership has been cancelled.") {
 		global $current_user;
-		$translated_text = "Your recurring subscription has been cancelled. Your active membership will expire on " . date(get_option("date_format"), $pmpro_next_payment_timestamp) . ".";
+		$translated_text = sprintf( __( 'Your recurring subscription has been cancelled. Your active membership will expire on %s', 'pmpro-cancel-on-next-payment-date' ), date( get_option( 'date_format' ), $pmpro_next_payment_timestamp ) );

```

Comment:
Please can you add a full stop at the end of the text here. (You may need to regenerate the pot files)